app/models/peep.rb

```diff
@@ -0,0 +1,11 @@
+class Peep
+  include DataMapper::Resource
+
```

Comment:
I'd include some validations here for things that must be in every Peep (and test for them in peep_spec.rb)